docs/testcases/UI/Dashboard/2356.md

```diff
@@ -0,0 +1,26 @@
+# Allow to configure several "support" icons
+
+Test verifies that support buttons with different icons and tooltip content can be rendered at the left panel for each user role/group.
+
+**Prerequisites**:
+- Admin user
+- Non-admin user
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the admin user from the prerequisites | |
+| 2 | Open the **Settings** page | |
+| 3 | Click the **PREFERENCES** tab | |
+| 4 | Click the **User Interface** tab | |
+| 5 | Find the **`ui.support.template`** preference | |
+| 6 | Save the preference value, then clear and input the following json <ul> `{` <ul> `"ROLE_ADMIN": {` <ul> `"icon1": {` <ul> `"icon": "rocket",` <br> `"content": "CONTENT1 for <br> Admin users"` </ul> `},` <br> `"icon2": {` <ul> `"icon": "https://t.alipayobjects.com/images/rmsweb/T1B9hfXcdvXXXXXXXX.svg",` <br> `"content": "CONTENT2 for <br> Admin users"` </ul> `}` </ul> `},` <br> `"_default": {` <ul> `"icon3": {` <ul> `"icon" : "key",` <br> `"content" : "CONTENT3 for <br> Non_admin users"` </ul> `}` </ul> `}` </ul> `}` | |
+| 7 | Click **Save** button | |
+| 8 | Refresh page | 2 *help* buttons with *icon1* and *icon2* specified in json at step 6 are shown at the bottom of the left panel | 
+| 9 | Click button with *icon1* | Text `CONTENT1 for Admin users` from json specified at step 6 is displayed on the *icon1* tooltip |
+| 10 | Click button with *icon2* | Text `CONTENT2 for Admin users` from json specified at step 6 is displayed on the *icon2* tooltip |
+| 11 | Logout | |
+| 12 | Login as the non-admin user from the prerequisites | 1 *help* button with *icon3* specified in json at step 6 is shown at the bottom of the left panel |
+| 13 | Click button with *icon3* | Text `CONTENT2 for Admin users` from json specified at step 6 is displayed on the support icon tooltip |
```

Comment:
Fix `CONTENT2 for Admin users` to `CONTENT3 for Non_admin users`

Reply:
Done